libs/ui/components/inputs/button-icon/index.stories.tsx

```diff
@@ -109,11 +109,11 @@ RemoveMemberSmall.args = {
     gap: '10px',
     backgroundColor: '#FFFFFF',
     hoverBackgroundColor: '#F2F4F9',
-    iconComponent: (isHovered) => (
+    iconComponent: ({ isHovered }) => (
         <Icon
             pointerEvents='none'
             name={'removeSmall'}
-            stroke={isHovered ? '#F46363' : '#8F95B2'}
+            stroke={{ isHovered } ? '#F46363' : '#8F95B2'}
```

Comment:
```suggestion
            stroke={ isHovered  ? '#F46363' : '#8F95B2'}
```
I don't know why you put it in an object

Reply:
I just forgot to change it after a "replace all"